src/views/auth/Signout.vue

```diff
@@ -6,14 +6,18 @@
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
 import SbcSignout from 'sbc-common-components/src/components/SbcSignout.vue'
+import { mapGetters } from 'vuex'
 
 @Component({
-  components: { SbcSignout }
+  components: { SbcSignout },
+  computed: {
+    ...mapGetters(['getLoginUrl'])
```

Comment:
Were you going to change these to `@Getter` syntax so we have typing?

Reply:
Yes, good catch, looks like I missed this one